cmd/create/accountroles/creators.go

```diff
@@ -384,6 +384,38 @@ func (hcp *hcpManagedPoliciesCreator) printCommands(r *rosa.Runtime, input *acco
 				return err
 			}
 
+			isHcpInstallerRole := role.Name == aws.HCPAccountRoles[aws.HCPInstallerRole].Name
+
+			if isHcpInstallerRole && input.isSharedVpc { // HCP shared VPC
+				// Route53 Policy
+				userProvidedRoleName, err := aws.GetResourceIdFromARN(args.route53RoleArn)
```

Comment:
I find that a loop may help in readability considering it does the same steps for both